migrations/versions/492839e0f6a_added_speaker_invites_table.py

```diff
@@ -0,0 +1,33 @@
+"""Added speaker_invites table
```

Comment:
This migration should be squashed into the first one. If doing that now messes up your progress, it's okay to hold off on squashing them until the end, but you probably don't want too many of them to accumulate before you do that.